paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -326,6 +326,7 @@ void CpuPassStrategy::EnableMKLDNN() {
              "softplus_activation_mkldnn_fuse_pass",  //
              "shuffle_channel_mkldnn_detect_pass",    //
              "elt_act_mkldnn_fuse_pass",              //
+             "operator_scale_onednn_fuse_pass",       //
```

Comment:
One more question - are we gonna use this pass only in FP32 models or in INT8 as well?

Reply:
We can use it in INT8 as well. For now, we will reuse approach with post-op like in FP32.